# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../evidence/evas_api.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- evidence/src/gui/gtk2/common/tooltip.c_orig	2006-11-05 15:54:34.053006000 +0100
+++ evidence/src/gui/gtk2/common/tooltip.c	2006-11-05 16:00:25.674981000 +0100
@@ -197,9 +197,9 @@
   if(e) {
     o=(Evas_Object *)e->data;         /* fade out */
     for(c=7;c>=0;c-=2) {
-      evas_object_gradient_colors_clear(o);
-      evas_object_gradient_color_add(o,255,255,255, c*4,     63);
-      evas_object_gradient_color_add(o,255,255,255, 31+c*32,  7);
+      evas_object_gradient_clear(o);
+      evas_object_gradient_color_stop_add(o,255,255,255, c*4,     63);
+      evas_object_gradient_color_stop_add(o,255,255,255, 31+c*32,  7);
       evas_render(evas); }}
   else
     E(1,"evas_motioned: no gradient in tooltip, skipping fade-out sequence...\n");
